Assets/BossRoom/Scripts/Server/Game/Action/DashAttackAction.cs

```diff
@@ -0,0 +1,85 @@
+using MLAPI;
```

Comment:
As a user, the first thing I did was press the dash without setting a target. I saw my character do some vfx, but nothing else. Simple screen message or setting a default target destination would help

Reply:
Added check so that if the teleport-destination seems unset (is precisely Vector3.zero) or is extremely close to the character's current location, we now pick a new spot that's ActionDescription.Range away. This ensures that clicking the action at startup has a noticeable effect.